build_tools/cmake/iree_cc_test.cmake

```diff
@@ -55,7 +60,7 @@ function(iree_cc_test)
     _RULE
     ""
     "NAME"
-    "SRCS;COPTS;DEFINES;LINKOPTS;DATA;DEPS;LABELS"
+    "ARGS;SRCS;COPTS;DEFINES;LINKOPTS;DATA;DEPS;LABELS"
```

Comment:
Add bazel->cmake support coming from cc_test?

Reply:
Thought you might ask for that :) Done.